google-cloud-storage/src/test/scala/docs/scaladsl/GCStorageSinkSpec.scala

```diff
@@ -41,7 +40,10 @@ class GCStorageSinkSpec
     val secondChunkContent = Random.alphanumeric.take(chunkSize).mkString
     val metadata = Map(Random.alphanumeric.take(5).mkString -> Random.alphanumeric.take(5).mkString)
 
-    mockLargeFileUpload(firstChunkContent, secondChunkContent, chunkSize, Some(metadata))
+    mock.simulate(
+      mockTokenApi,
+      mockLargeFileUpload(firstChunkContent, secondChunkContent, chunkSize, Some(metadata))
+    )
```

Comment:
Curious why this changed.

Reply:
Why this specific test changed, or why I had to redo all the GCS tests? I couldn't figure out how to get WireMock to work with the forward proxy so I rewrote them with Hoverfly. If someone knows how to configure this I'd love to revert these back to the original WireMock tests.

Reply:
Got it. I don't have any particular attachment to WireMock. As long as the same interactions are recorded/replayed it should be fine.

Reply:
The WireMock and Hoverfly DSLs are very similar so it was a straightforward change.